site/src/styles/global.css

```
@@ -26,291 +13,37 @@ html,
-    radial-gradient(circle at 15% 10%, #e8f1ff 0%, transparent 40%),
-    linear-gradient(180deg, #fafdff 0%, var(--bg) 45%, #eef5ff 100%);
-  font-family: 'IBM Plex Sans', 'Avenir Next', 'Segoe UI', sans-serif;
-}
-
-main {
-  width: min(1120px, 92vw);
-  margin: 0 auto;
-  padding: 2rem 0 4rem;
+  color: #131b2e;
+  background: #faf8ff;
+  font-family: 'Inter Variable', 'Inter', system-ui, sans-serif;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)